src/Sarif.UnitTests/SarifCurrentToVersionOneTests.cs

```diff
@@ -1963,5 +1963,286 @@ public void SarifTransformerTests_ToVersionOne_Regions()
 
             VerifyCurrentToVersionOneTransformation(V2LogText, V1LogExpectedText);
         }
+
+        [Fact]
+        public void SarifTransformerTests_ToVersionOne_CodeFlows()
+        {
+            const string V2LogText =
+@"{
+  ""$schema"": ""http://json.schemastore.org/sarif-2.0.0"",
+  ""version"": ""2.0.0"",
+  ""runs"": [
+    {
+      ""tool"": {
+        ""name"": ""CodeScanner"",
+        ""semanticVersion"": ""2.1.0""
+      },
+      ""logicalLocations"": {
```

Comment:
You don't need `logicalLocations` to test code flows.

Reply:
Removed